.build/images/dietpi-installer

```diff
@@ -574,14 +575,15 @@ _EOF_
 				G_EXEC sed -i 's/coherent_pool=2M/coherent_pool=2M usbcore.autosuspend=-1/' /boot/boot.cmd
 
 			# Rockchip 64-bit (configs work with Amlogic OOTB)
-			elif [[ $G_HW_MODEL =~ ^(42|43|46|47|55|56|58|68|72|73|77|78)$ ]]
+			elif [[ $G_HW_MODEL =~ ^(42|43|46|47|55|56|58|68|72|73|77|78|80)$ ]]
 			then
 				G_EXEC sed -Ei '/^setenv (kernel|fdt)_addr_r/d' /boot/boot.cmd
 				G_CONFIG_INJECT 'setenv scriptaddr ' 'setenv scriptaddr "0x9000000"' /boot/boot.cmd
 				G_CONFIG_INJECT 'overlay_path=' 'overlay_path=rockchip' /boot/dietpiEnv.txt
 				case $G_HW_MODEL in
 					77) G_CONFIG_INJECT 'overlay_prefix=' 'overlay_prefix=rk35xx' /boot/dietpiEnv.txt;;
 					78) G_CONFIG_INJECT 'overlay_prefix=' 'overlay_prefix=rk3588' /boot/dietpiEnv.txt;; # ToDo: There are multiple other prefixes used in the kernel package, "rock-5b", "rock-5ab", "rockchip" (for fixup), so the boot.cmd need heavy adjustments (eliminate the prefix entirely) to work with all overlays.
+					80) G_CONFIG_INJECT 'overlay_prefix=' 'overlay_prefix=rockchip-rk3588' /boot/dietpiEnv.txt;;
```

Comment:
Since all related overlay files in `/boot/dtb/rockchip/overlay/` do indeed have the `-opi5` in their filename as well. This way it's little easier/shorter to add overlays, and it is assured that no non-matching are applied.
```suggestion
					80) G_CONFIG_INJECT 'overlay_prefix=' 'overlay_prefix=rockchip-rk3588-opi5' /boot/dietpiEnv.txt;;
```
I'm actually wondering whether the `rk3588-` prefixed ones work as well. Do you have some I2C/PWM/SPI/UART device to test? In case it it the same problem as with ROCK Pi 5B that one cannot support all overlays with any non-empty prefix.

Reply:
Yes, I may be able to test later.